src/qsub.jl

```diff
@@ -68,7 +70,7 @@ function launch(manager::Union{PBSManager, SGEManager}, params::Dict, instances_
 end
 
 function manage(manager::Union{PBSManager, SGEManager}, id::Integer, config::WorkerConfig, op::Symbol)
-    if op == :finalize
+    if op == :interrupt
```

Comment:
Using ```rmprocs(workers())``` with the current implementation of qsub (with ```:finalize``` to kill workers) returns ```:ok``` but when you Ctrl+D to exit julia also prints to STDOUT unending stream of commands and must be killed by hand.

I tried implementing 

```
function  kill(manager::Union{PBSManager, SGEManager}, id::Integer, config::WorkerConfig)
    kill(config.userdata[:process])
end
 ```

And the same happens with ```:finalinze```.

If you look in managers.jl, there the killing of workers is never handled in a ```:finalize``` op, always in an ```:interrupt```. Finalizers just show up in multi.jl:

```
 # install a finalizer to perform cleanup if necessary
finalizer(w, (w)->if myid() == 1 manage(w.manager, w.id, w.config, :finalize) end)
```

So we could follow this model and keep my implementation of the ```manage``` function as is, which is the only thing currently working for me, and do any cleanup in a finalize. What do you think?